src/ILCompiler.ReadyToRun/src/ILCompiler.ReadyToRun.csproj

```diff
@@ -48,9 +50,39 @@
     <Compile Include="ObjectWriter\SectionBuilder.cs" />
     <Compile Include="ObjectWriter\R2RPEBuilder.cs" />
     <Compile Include="ObjectWriter\RelocationHelper.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
     <Compile Include="..\..\Common\src\System\Collections\Generic\ArrayBuilder.cs">
       <Link>Common\ArrayBuilder.cs</Link>
     </Compile>
+    <Compile Include="..\..\Common\src\Internal\Runtime\EEType.Constants.cs">
+      <Link>Common\EEType.Constants.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Common\src\Internal\Runtime\EETypeBuilderHelpers.cs">
+      <Link>Common\EEType.cs</Link>
+    </Compile>
+    <Compile Include="..\..\ILCompiler.Compiler\src\Compiler\JitHelper.cs">
```

Comment:
EEType.Constants, EETypeBuilderHelpers, JitHelper are CoreRT specific - could you mark this inclusion as a candidate for further refactoring later?

Reply:
After factoring out `RequiresAlign8` I'm just left with JitHelper.cs. I left a note to refactor it later.